platform/src/main/java/org/stellar/anchor/platform/utils/TransactionHelper.java

```diff
@@ -81,6 +93,18 @@ static GetTransactionResponse toGetTransactionResponse(JdbcSep24Transaction txn)
         .build();
   }
 
+  private static String makeAsset(
+      @Nullable String dbAsset, AssetService service, JdbcSep24Transaction txn) {
+    if (dbAsset != null) {
+      return dbAsset;
+    }
+
+    AssetInfo info = service.getAsset(txn.getRequestAssetCode(), txn.getRequestAssetIssuer());
```

Comment:
There shouldn't be any function called `getRequestAssetCode()` or `getRequestAssetIssuer()`

Reply:
Why not? It's what we store in DB

Reply:
This field already exists in the DB? How do we use it today if not for the amount expected?

Reply:
Used for refunds

Reply:
Ok, then I'll resolve this. We may want to include a migration to a column to the same name eventually though.

Reply:
I guess we can store expectedAsset instead